tools/wptrunner/wptrunner/wpttest.py

```diff
@@ -292,6 +293,8 @@ def disabled(self, subtest=None):
 
     @property
     def restart_after(self):
+        if self.restart_test_after:
+            return True
```

Comment:
How about delete line 296-297, and change line 302 to `return restart_test_after`?

Reply:
Ack